interface/reports/report.script.php

```diff
@@ -236,6 +236,10 @@ function appendOptionDateCriteria(text0, value0, text1, value1, seperator, FromD
         elOptNew.text = text0 + seperator + text1;
         FromDateValue = document.getElementById(FromDate).value;
         ToDateValue = document.getElementById(ToDate).value;
+        // save a few clicks
+        if (ToDateValue == '') {
+            ToDateValue = FromDateValue;
+        } 
```

Comment:
This could be refactored to `ToDateValue = (ToDateValue == '') ? FromDateValue : ToDateValue;` which is more concise and easier to read

Reply:
i am not sure i would consider this easier to read. I took me 15 seconds of doing reverse logic in my head to confirm this was the same as the original simpleton if above :) I still generally use simpleton if/else blocks in my coding; for some reason the Ternary Operator throws me unless it's super simple.

Reply:
To clarify, I am ok with use of Ternary Operator (including here) since it definitely has its many uses. Just making point that in my brain, at least, readability is not one of its strong suits :)